README.md

```diff
@@ -64,6 +64,7 @@ Learn more about [architecture](https://docs.saleor.io/docs/3.x/overview/archite
 
 - **Headless / API first**: Build mobile apps, custom storefronts, POS, automation, etc
 - **Extensible**: Build anything with webhooks, apps, metadata, and attributes
+- **App Store**: Leverage a collection of built-in integrations
```

Comment:
Question: As Apps are not embedded in Saleor core, wouldn't be good to provide here a link with more details related to Apps? Maybe this one: https://github.com/saleor/apps ?